client/my-sites/email/titan-setup-thank-you/theme.ts

```diff
@@ -0,0 +1,23 @@
+export type ThankYouThemeProps = { theme?: ThankYouThemeType };
+export type ThankYouThemeType = {
+	colors: {
+		backgroundColorHeader: string;
+		textHeaderColor: string;
```

Comment:
Can we rename these to `headerBackgroundColor` and `headerTextColor`? This keeps the naming consistent and makes it easier to start from the element you're interested in.